src/App.js

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import "./styles.css";
+import Stars from "./Stars";
+
+const sections = {
+  ALL: "All",
+  CORE: "Core Framework",
+  PLUGIN: "Plugins",
+  OTHER: "Other Packages"
+};
+
+const docs = [
+  {
+    name: "Ape Framework",
+    link: "https://docs.apeworx.io/ape/stable/index.html",
```

Comment:
technically... this should work
```suggestion
    link: "./ape",
```

Reply:
as discussed in tg this would break local build so will keep absolute link